features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -153,7 +150,20 @@ FileHandle *ATHandler::get_file_handle()
 
 void ATHandler::set_file_handle(FileHandle *fh)
 {
+    _fh_sigio_set = false;
```

Comment:
If _fileHandle already has sigio set that should be removed?

Reply:
Answering my own question above. We have internal ticket IOTCELL-846 for that. This PR looks good.